{"name": "Проверка отображения существующего заказа из истории пользователя в ленте", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <test_feed_page.TestFeedPage object at 0x107276e40>, driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"e3335163-33b1-4154-bb82-c0d9ef7103ce\")>\ncreate_user_and_order_and_delete = ('Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY2ZTczMDVhOWVkMjgwMDAxYjRiYTA5NSIsImlhdCI6MTcyNjQyNzIyNiwiZXhwIjoxNzI2NDI4NDI2fQ.2xYPK6gSfvYNlQOFt1c56hoa_3CA8vXbVbZJkdWySiU', <Response [200]>)\nset_user_tokens = None\n\n    @allure.title('Проверка отображения существующего заказа из истории пользователя в ленте')\n    def test_displaying_in_feed_new_order_from_history_success(self, driver, create_user_and_order_and_delete,\n                                                               set_user_tokens):\n        main_page = MainPage(driver)\n        account_page = AccountPage(driver)\n        order_history_page = OrderHistoryPage(driver)\n        feed_page = FeedPage(driver)\n        main_page.click_on_personal_account_in_header()\n>       account_page.click_on_order_history_button()\n\ntests/test_feed_page.py:27: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage_objects/account_page.py:9: in click_on_order_history_button\n    self.click_on_element(AccountPageLocators.order_history)\npage_objects/base_page.py:22: in click_on_element\n    target = self.check_element_is_clickable(locator)\npage_objects/base_page.py:49: in check_element_is_clickable\n    return WebDriverWait(self.driver, 15).until(expected_conditions.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e3335163-33b1-4154-bb82-c0d9ef7103ce\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x1072f58a0>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException"}, "steps": [{"name": "Кликнуть по кнопке перехода в личный кабинет в хэдере", "status": "passed", "steps": [{"name": "Подождать прогрузки элемента", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', '//p[text()=\"Личный Кабинет\"]/parent::a')"}], "start": 1726427228508, "stop": 1726427228518}, {"name": "Кликнуть на элемент", "status": "passed", "steps": [{"name": "Проверить кликабельность элемента", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', '//p[text()=\"Личный Кабинет\"]/parent::a')"}], "start": 1726427228518, "stop": 1726427228530}], "parameters": [{"name": "locator", "value": "('xpath', '//p[text()=\"Личный Кабинет\"]/parent::a')"}], "start": 1726427228518, "stop": 1726427228789}], "start": 1726427228508, "stop": 1726427228789}, {"name": "Кликнуть по кнопке \"История заказов\"", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/account_page.py\", line 9, in click_on_order_history_button\n    self.click_on_element(AccountPageLocators.order_history)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/base_page.py\", line 22, in click_on_element\n    target = self.check_element_is_clickable(locator)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/base_page.py\", line 49, in check_element_is_clickable\n    return WebDriverWait(self.driver, 15).until(expected_conditions.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Кликнуть на элемент", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/base_page.py\", line 22, in click_on_element\n    target = self.check_element_is_clickable(locator)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/base_page.py\", line 49, in check_element_is_clickable\n    return WebDriverWait(self.driver, 15).until(expected_conditions.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Проверить кликабельность элемента", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/kosova.ea/Diplom/Diplom_3/page_objects/base_page.py\", line 49, in check_element_is_clickable\n    return WebDriverWait(self.driver, 15).until(expected_conditions.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "parameters": [{"name": "locator", "value": "('xpath', '//a[@href = \"/account/order-history\"]')"}], "start": 1726427228789, "stop": 1726427244040}], "parameters": [{"name": "locator", "value": "('xpath', '//a[@href = \"/account/order-history\"]')"}], "start": 1726427228789, "stop": 1726427244041}], "start": 1726427228789, "stop": 1726427244041}], "parameters": [{"name": "driver", "value": "<class 'selenium.webdriver.firefox.webdriver.WebDriver'>"}], "start": 1726427228507, "stop": 1726427244042, "uuid": "9c66e37b-46eb-4463-a4bf-672b97e2845d", "historyId": "5f763717bdea65d15b883adcfb7e6173", "testCaseId": "7dae3a945fa8ea5d8cb9585abd852a14", "fullName": "tests.test_feed_page.TestFeedPage#test_displaying_in_feed_new_order_from_history_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_feed_page"}, {"name": "subSuite", "value": "TestFeedPage"}, {"name": "host", "value": "NB-ISC-69172.local"}, {"name": "thread", "value": "15735-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_feed_page"}]}